### PR TITLE
TDI-38197: Performance issue for tMssqlOutput.

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-38197

### CODE REVIEW

**Comment by pyzhou-talend** [comment]
The inner class disable the branch clean. So after first batch, it will commit on every insert. And here's another issue, it will create 10,000 instance of the inner class.

File: main/plugins/org.talend.designer.components.localprovider/components/tMSSqlOutput/tMSSqlOutput_main.javajet [hunk 1]
```diff
@@ -1558,7 +1558,6 @@ if((cid.equals("talendLogs_DB") || cid.equals("talendStats_DB") || cid.equals("t
             	class LimitBytesHelper_<%=cid%>{
             		public void limitBytePart1(int batchSize_<%=cid%>,int batchSizeCounter_<%=cid%>,int insertedCount_<%=cid%>,
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMSSqlOutput/tMSSqlOutput_main.javajet]: A question not about your modification: Could you check the NB line value about this component?
I just think they are invalid when commit split into batches.

**Comment by jzhao-talend** [comment]
@pteyssier Because of this issue is caused by #540
I found another issue about this PR. So I prefer to recheck #540 and revert wrong part.
(changes about batch part)

File: main/plugins/org.talend.designer.components.localprovider/components/tMSSqlOutput/tMSSqlOutput_main.javajet (hunk 1 above)
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMSSqlOutput/tMSSqlOutput_main.javajet]: Confirmed with simple test. The nb line would be invalid, if it included several batches.

**Pull request merged**